### DISCUSSION TITLE
Took Monika out today, when I got back I returned her file to the characters folder. But upon opening MAS, She was still out of the room

### DISCUSSION THREAD

**Discussion by Doriftobowler** [Monika-After-Story/MonikaModDev#9624] [QA - tech support + general] [answered]
![image](https://user-images.githubusercontent.com/114362352/192163440-a9133338-7432-4b50-9876-fc89ece217e0.png)

What happened and what can I do about this?

**Comment by Booplicate** [comment, marked as answer]
Her file got corrupted, restore a backup (of the file, or [her persistent](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent), anything you have).

**Reply by Doriftobowler** [reply]
I restored different backups of the Monika file and only the one I deleted that morning itself worked. Thank you
